lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -369,6 +374,12 @@ def configure(conf)
                   ' partial_success flag will be ignored.'
       end
 
+      unless @logging_api_url == DEFAULT_LOGGING_API_URL || @use_grpc
+        @log.warn 'Detected customized logging_api_url while use_grpc is not' \
+                  ' enabled. Customized logging_api_url for the non-gRPC path' \
+                  ' is not supported. The config will be ignored.'
```

Comment:
`The logging_api_url option will be ignored.`.

Reply:
Added.